### ISSUE TITLE
Adding a package.json

### ISSUE THREAD

**Issue by ztl8702** [mmig/libflac.js#8]
## TL;DR
Please add a `package.json` to the root of the repository so that libflac.js can be used in other JS-based projects more easily using npm.

---
Hi, thanks for making this library. I am currently using libflac.js in [Jitsi Meet](https://github.com/jitsi/jitsi-meet) to encode recorded audio in the browser. 

Because of the set up of the project, it is better if we can reference libflac.js as a npm package. The only missing part for that to happen is a `package.json` file in the root of the repository. Currently, I am maintaining a fork, but the only modification is the addition of `package.json`. I think it is better if this is done by you guys.

I have created a simple pull request #7 and filled in the file for you. You don't have to use my PR.

You don't necessarily need to publish libflac.js to npmjs.com, because npm can treat git repository as a package as long as there is a `package.json` file in the root. Just adding a package.json file can make it much easier for other projects to use libflac.js.

**Comment by ztl8702** [comment]
@russaa
